tests/CarbonSleeper/SleepTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Tests\Carbon;
+
+/*
+ * This file is part of the Carbon package.
+ *
+ * (c) Brian Nesbitt <brian@nesbot.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Carbon\Carbon;
+use Carbon\CarbonSleeper;
+use Tests\AbstractTestCase;
+
+class SleepTest extends AbstractTestCase
+{
+    protected function tearDown()
+    {
+        Carbon::setTestNow();
+        parent::tearDown();
+    }
+
+    public function testFakeSleepDelay()
+    {
+        Carbon::setTestNow(Carbon::now());
+
+        $before = time();
+
+        $carbonBefore = Carbon::now();
+
+        CarbonSleeper::sleep(5);
+
+        $after = time();
+
+        $carbonAfter = Carbon::now();
+
+        $this->assertLessThanOrEqual(
+            1, // Allow up to 1 second in case of timing issue between calls
+            $after - $before,
+            'Carbon sleep called system sleep in test condition'
```

Comment:
We prefer all tests to be the same, and the testFunction name is self-explanatory